pkg/kubelet/server/server_test.go

```diff
@@ -230,7 +230,8 @@ func newServerTest() *serverTestFramework {
 		stats.NewResourceAnalyzer(fw.fakeKubelet, time.Minute, &kubecontainertesting.FakeRuntime{}),
 		fw.fakeAuth,
 		true,
-		&kubecontainertesting.Mock{})
+		&kubecontainertesting.Mock{},
+		nil)
```

Comment:
Set a non-nil here to make sure all the wiring happens correctly when the cri handler is registered